pulp_2_tests/tests/rpm/api_v2/test_modularity.py

```diff
@@ -55,3 +73,22 @@ def test_sync_publish_repo(self):
             'modules',
             api.safe_handler,
         )
+
+    @skip_if(bool, 'repos', False)
```

Comment:
Try to decouple the tests. Even if it will be necessary to re-create functions to achieve such.

For instance, create a function, `create_repos()` that can be share within the tests.